packages/@glimmer/core/src/owner.ts

```diff
@@ -0,0 +1,27 @@
+import { DEBUG } from '@glimmer/env';
+
+const OWNER_MAP = new WeakMap<{}, unknown>();
+
+export const OWNER_KEY = `__OWNER_${Math.floor(Math.random() * Date.now())}__`;
+
+export let DEFAULT_OWNER = {};
```

Comment:
I'm curious here, do you think this `DEFAULT_OWNER` solution is overall better than just passing `null`? I think the errors will be really nice with the current implementation, just wondering if you had considered "just pass null"...

Reply:
The main reason I did it this way was so that manager instances could go into the `WeakMap` mapped to owners. If we allowed `null`, we would have to have a branch for that I suppose